docs/user-quick-start.md

```diff
@@ -0,0 +1,299 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quick start guide for testing application DR
+
+This page will help you to set up an environment for testing **Regional-DR** basic tests.
```

Comment:
I don't think we need double `**` - one is enough to emphasize.

Reply:
Not a problem, just not consistent with the developer quick start guide, we use *this* to emphasize.